server/migrations/20210722090328_create_plan_area_changes_table.js

```diff
@@ -0,0 +1,17 @@
+exports.up = function(knex) {
+    return knex.schema.createTableIfNotExists('plan_area_changes', t => {
+        t.increments('id').primary();
+        t.integer('plan_id');
+        t.text('usage');
+        t.text('measurement_unit').notNullable();
+        t.text('approved_state').notNullable();
```

Comment:
Leave them nullable, data will not be complete

Reply:
Right now, it is complete for every plan that we got.